controller/node_controller.go

```diff
@@ -355,13 +356,22 @@ func (nc *NodeController) syncDiskStatus(node *longhorn.Node) error {
 			diskStatus.StorageAvailable = diskInfo.StorageAvailable
 		}
 
+		condition := types.GetNodeCondition(diskConditions, types.NodeConditionTypeDiskSchedulable)
```

Comment:
Need to use the new type `DiskConditionType` and `DiskConditionTypeSchedulable`.

Those conditions cannot be set on the node level. So they should get a new type instead of reusing the node ones.